### PR TITLE
don't load the settings during function eval.

### PR DESCRIPTION
**Pull request by aronasorman**
This causes a circular import where a settings module imports this
module at the toplevel. Doing that import forces django to load the
LazySettings attributes, which causes an import of all settings modules,
which causes this module to get loaded, which forces django to load the
LazySettings attributes...

Loading settings at actual function call avoids this circular import.

### CODE REVIEW

**Comment by rtibbles** [comment]
Yes, this looks sensible.

File: kalite/i18n/__init__.py [hunk 1]
```diff
@@ -163,12 +163,16 @@ def get_id2oklang_map(video_id, force=False):
...
-def get_youtube_id(video_id, lang_code=settings.LANGUAGE_CODE):
+def get_youtube_id(video_id, lang_code=None):
     """Given a video ID, return the youtube ID for the given language.
     If lang_code is None, return the base / default youtube_id for the given video_id.
     If youtube_id for the given lang_code is not found, function returns None.
     Accepts lang_code in ietf format
     """
+
+    if not lang_code:
+        lang_code = settings.LANGUAGE_CODE
```
Comment (benjaoming) [inline kalite/i18n/__init__.py]: If the problem is caused by loading settings inside the __init__ module, you should load it inside the function body instead?

Comment (aronasorman) [inline kalite/i18n/__init__.py]: By settings you mean just importing `django.conf.settings`? I believe under the hood it imports them lazily (not sure, haven't read the source), so it only actually imports settings when you access an attribute.

Comment (benjaoming) [inline kalite/i18n/__init__.py]: Ah! That could be, I lack knowledge about this, but it could explain why certain things actually work! :)

Comment (benjaoming) [inline kalite/i18n/__init__.py]: True, it won't be loaded until you fetch an attribute!

**Pull request merged**